onward/app/feed/MostViewedVideoAgent.scala

```diff
@@ -43,13 +40,14 @@ class MostViewedVideoAgent(contentApiClient: ContentApiClient, ophanApi: OphanAp
       val mostViewed: Future[Seq[Video]] = contentApiClient.getResponse(contentApiClient.search(Uk)
         .ids(contentIds)
         .pageSize(20)
-      ).map{ r =>
+      ).map { r =>
         val videoContent: Seq[client.model.v1.Content] = r.results.filter(_.isVideo)
         log.info(s"Number of video content items from CAPI: ${videoContent.size}")
         videoContent.map(Content(_)).collect { case v: Video => v }
       }
 
       mostViewed.filter(_.nonEmpty).foreach(agent.alter)
+      mostViewed
```

Comment:
Could we do some magic here with `map` and `Future.sequence` instead of just returning `mostViewed`?  Probably nicer to return the nonEmpty ones anyway.